### PR TITLE
[release-1.19] updated perf test from CNCF infra

### PR DESCRIPTION
**Pull request by Shuayb1**
## Reviewers

- [ ] Ambient
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ X] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by ericvn** [comment]
Noting that this is in 1.19 data and currently we won't see this actually get reflected in the docs unless we rebuild the 1.19 archive after this PR merges. I'm fine with that, just need to verify the steps. Also noting there is discussion about doing the archives differently.

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 1]
```diff
@@ -105,21 +105,25 @@ average and tail latencies. The actual tail latency depends on the traffic patte
...
 Inside the mesh, a request traverses the client-side proxy and then the server-side
 proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
-the two proxies add about 1.31 ms and 1.58 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+the two proxies add about 0.175 ms and 0.235 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
 We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
 for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 16 client connections, 2 proxy workers and mutual TLS enabled.
 
-{{< image width="90%"
-    link="latency_p90_fortio_with_jitter.svg"
+Note: This testing was performed on the [CNCF Community Infrastructure Lab](https://github.com/cncf/cluster).
```
Comment (AndreaM12345) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:112]: do we want to note that different environments provide different performance results?

**Comment by linsun** [comment]
> Noting that this is in 1.19 data and currently we won't see this actually get reflected in the docs unless we rebuild the 1.19 archive after this PR merges. I'm fine with that, just need to verify the steps. Also noting there is discussion about doing the archives differently.

Thanks @ericvn for pointing it out. Do we have documented steps to do that?

**Review by linsun — APPROVED** [review]
Thank you very much @Shuayb1!

**Review by craigbox — APPROVED** [review]

**Pull request merged**